src/pages/basic/MentalHealthTips.jsx

```diff
@@ -1,13 +1,71 @@
-import React from 'react';
-import Button from '../../components/common/button';
-import { healthTips } from '../../data';
+import React, { useState } from "react";
```

Comment:
you can add your useState import to where we import React on the first line.
`import React, {.useState } from 'react'` instead of importing them on separate lines.